ChatRPG/API/OpenAiGptMessage.cs

```diff
@@ -21,9 +22,15 @@ public OpenAiGptMessage(ChatMessageRole role, string content)
         }
     }
 
+    public OpenAiGptMessage(ChatMessageRole role, string content, UserPromptType userPromptType) : this(role, content)
+    {
+        UserPromptType = userPromptType;
+    }
+
     public ChatMessageRole Role { get; }
     public string Content { get; private set; }
     public string NarrativePart { get; private set; }
+    public readonly UserPromptType UserPromptType = UserPromptType.Do;
```

Comment:
Synes det lyder lidt weird at kalde både typen og navnet for UserPromptType

Reply:
Ja, men kan ikke lige finde på nogle alternativer.